server/sonar-web/src/main/js/apps/global-permissions/users-view.js

```diff
@@ -22,7 +22,7 @@ import Template from './templates/global-permissions-users.hbs';
 import '../../components/SelectList';
 
 function getSearchUrl (permission, project) {
-  let url = '/api/permissions/users?ps=100&permission=' + permission;
+  let url = window.baseUrl + '/api/permissions/users?ps=100&permission=' + permission;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)